script.go

```diff
@@ -11,6 +11,8 @@ import (
 type Scripter interface {
 	Eval(ctx context.Context, script string, keys []string, args ...interface{}) *Cmd
 	EvalSha(ctx context.Context, sha1 string, keys []string, args ...interface{}) *Cmd
+	EvalRO(ctx context.Context, script string, keys []string, args ...interface{}) *Cmd
+	EvalShaRO(ctx context.Context, sha1 string, keys []string, args ...interface{}) *Cmd
```

Comment:
I checked interface changes for `Scripter` and `Cmdable` don't affect implementation within go-redis, but changing interfaces can be always affect its implementations...

Reply:
Thanks, this looks safe enough especially for v8.